server.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
```

Comment:
This is exactly what I like to see for a server file.  clean and easy to read, routes broken out into their own file.  Awesome job.